fastscanner/services/indicators/ports.py

```diff
@@ -50,6 +50,8 @@ async def push(self, channel_id: str, data: dict[Any, Any], flush: bool = True):
 
     async def flush(self): ...
 
+    async def redis_cleanup(self): ...
```

Comment:
The method can't be called redis_cleanup. The channel is an abstract contract. This is already forcing the contract to be implemented ONLY by Redis. If we implement a Channel called RabbitMQChannel, would it make sense it have a redis_cleanup method?